utils/dependencies.rb

```diff
@@ -45,4 +45,6 @@ def load_loader(loader_path)
 Dir[File.dirname(__FILE__) + '/../utils/**/base.rb'].each { |file| require_relative file }
 Dir[File.dirname(__FILE__) + '/../utils/**/*.rb'].each { |file| require_relative file }
 load_loaders
+
+Dotenv.overload('templates/.env.example')
 Dotenv.overload
```

Comment:
We can just pass `templates/.env.example` and `.env` as arguments for `Dotenv.overload`, like this: `Dotenv.overload('.env', 'templates/.env.example')`. The values from the first argument will have precedence

Reply:
I made it work here, but the second argument seems to have precedence instead.

Reply:
:rock: :rocket: :100: